dd-java-agent/instrumentation/jdbc/src/main/java/datadog/trace/instrumentation/jdbc/JDBCUtils.java

```diff
@@ -1,12 +1,31 @@
+// Modified by SignalFx
 package datadog.trace.instrumentation.jdbc;
 
 import datadog.trace.bootstrap.ExceptionLogger;
 import java.lang.reflect.Field;
 import java.sql.Connection;
 import java.sql.Statement;
+import java.util.regex.Pattern;
 
 public abstract class JDBCUtils {
   private static Field c3poField = null;
+  private static final String[] SqlNormalizeClauses = {
+    // Numerics
+    "0x[0-9a-fA-F]+", // hex
+    "(?<![\\p{LC}_0-9])[+.-]*[0-9]+[0-9xEe.+-]*", // regular numeric literal
```

Comment:
What's the significance of this lookbehind?

Reply:
It's what prevents FIELD2 identifiers from becoming FIELD?.